/ros2_laser_scan_merger:
  ros__parameters:
    laser1Alpha: 0.0
    laser1AngleMax: 181.0
    laser1AngleMin: -181.0
    laser1B: 0
    laser1G: 0
    laser1R: 255
    laser1XOff: 0.0
    laser1YOff: 0.0
    laser1ZOff: 0.0
    laser2Alpha: 0.0
    laser2AngleMax: 181.0
    laser2AngleMin: -181.0
    laser2B: 255
    laser2G: 0
    laser2R: 0
    laser2XOff: 0.0
    laser2YOff: 0.0
    laser2ZOff: 0.0
    pointCloudTopic: cloud_in
    pointCloutFrameId: laser
    qos_overrides:
      /parameter_events:
        publisher:
          depth: 1000
          durability: volatile
          history: keep_last
          reliability: reliable
    scanTopic1: /lidar_front_right/scan
    scanTopic2: /lidar_rear_left/scan
    show1: true
    show2: true
    use_sim_time: false

/pointcloud_to_laserscan:
  ros__parameters:
    angle_increment: 0.0087
    angle_max: 3.1415926
    angle_min: -3.1415926
    inf_epsilon: 1.0
    max_height: 1.0
    min_height: 0.0
    qos_overrides:
      /parameter_events:
        publisher:
          depth: 1000
          durability: volatile
          history: keep_last
          reliability: reliable
    queue_size: 16
    range_max: 4.0
    range_min: 0.45
    scan_time: 0.3333
    target_frame: laser
    transform_tolerance: 0.01
    use_inf: true
    use_sim_time: false